### PR TITLE
Set FamilyID, ExtendedProdID through macro, reset to default values  if fields absent in conf file

### PR DESCRIPTION
**Pull request by manojrupireddy**
https://github.com/openenclave/openenclave/issues/4112
Signed-off-by: manoj rupireddy <marupire@microsoft.com>

### CODE REVIEW

File: docs/GettingStartedDocs/buildandsign.md [hunk 1]
```diff
@@ -120,14 +120,17 @@ FamilyID=47183823-2574-4bfd-b411-99ed177d3e43
 ExtendedProductID=2768c720-1e28-11eb-adc1-0242ac120002
 ```
 
-As a convenience, you can specify the enclave properties in code using the `OE_SET_ENCLAVE_SGX2` macro to leverage SGX2 properties and enable KSS using RequireKSS argument in the macro. For example, the equivalent properties could be defined in any .c or .cpp file compiled into the enclave:
+On SGX2 machines, developers can also specify wether to handle in-enclave exceptions or not using `CapturePFGPExceptions` field.
```
Comment (RRathna) [inline docs/GettingStartedDocs/buildandsign.md:123]:
```suggestion
On SGX2 machines, developers can also specify whether to handle in-enclave exceptions or not using `CapturePFGPExceptions` field.
```

**Review by RRathna — COMMENTED** [review]
Hi Manoj,  can you also edit oesign's README.md and help text to mention the default values for FamilyID and ExtendedProductID? - https://github.com/openenclave/openenclave/blob/ffb7a0f574007f601780cfa023c63497c1ba6719/tools/oesign/main.c#L99-L103

File: docs/GettingStartedDocs/buildandsign.md (hunk 1 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/buildandsign.md]: @manojrupireddy , would it be possible to update oesign's README as @RRathna has stated above. After that, I can merge this.

**Review by radhikaj — APPROVED** [review]
bors r+

**Pull request merged**